meal_mate/app/src/main/java/com/android/engineer/mealmate/view/utils/constants/sealed_constants/AuthScreen.kt

```diff
@@ -0,0 +1,5 @@
+package com.android.engineer.mealmate.view.utils.constants.sealed_constants
```

Comment:
vague package name `sealed_constants`, not defining anything

Reply:
Got it, moved based on today's discussion.